client/src/components/ContractList/ContractList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './ContractList.css';
+// import {css,jsx} from '@emotion/core';
+
+
+const ContractList = () => {
+
+  let z = -1;
+
+
+  const contracts =
+    [{ title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' }, { title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' },{ title: 'My Contract' }]
+
+
+
+  const view = contracts.map(contract => {
+    z += 1;
```

Comment:
delete "z" and use this instead:
```
contracts.map((contract, index) => {
...
}
```
https://stackoverflow.com/questions/38364400/index-inside-map-function

Reply:
also, if it's an index, why not call it `i` or `index`?